common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -378,11 +374,7 @@ class Backend(
                         }
                     }
                 } else {
-                    synchronized(this@Backend) {
-                        identifyCallbacks.remove(cacheKey)
-                    }?.forEach { (_, onErrorHandler) ->
-                        onError(result.toPurchasesError().also { errorLog(it) })
```

Comment:
This was the bug 😮‍💨 

I should have called `onErrorHandler` instead of `onError`